### PR TITLE
02 feat: DHIS2-9648 show user message when they land in the registration page without org unit selected

### PR DESCRIPTION
**Pull request by paschalidi**
Hey @JoakimSM worked on this https://jira.dhis2.org/browse/DHIS2-9648

### What is this PR about?
Created a new component called `InefficientSelectionsMessage`. The component will show a message to the user where their selections are incomplete. See images below 👇 

### How this looks now? 
image
image
image

### CODE REVIEW

**Comment by paschalidi** [comment]
@cooper-joe perhaps you also wanna have a look

File: i18n/en.pot [hunk 1]
```diff
@@ -524,6 +524,9 @@ msgstr ""
 msgid "Completed"
 msgstr ""
 
+msgid "Choose in which organisation unit you want to start creating"
```
Comment (cooper-joe) [inline i18n/en.pot:527]: I suggest "Choose a registering unit to start reporting" here.

File: src/core_modules/capture-core/components/Pages/NewEvent/SelectionsIncomplete/DataEntrySelectionsIncomplete.component.js [hunk 2]
```diff
@@ -1,132 +1,52 @@
...
-    }
-    getText() {
-        let text;
-        const { isProgramSelected, isOrgUnitSelected } = this.props;
 
+const DataEntrySelectionsIncompletePlain = ({ classes, onCancel, isProgramSelected, isOrgUnitSelected }) => {
+    const getText = () => {
         if (!isProgramSelected && !isOrgUnitSelected) {
-            text = i18n.t('Select a registering unit and program above to get started');
+            return i18n.t('Select a registering unit and program above to get started');
```
Comment (cooper-joe) [inline src/core_modules/capture-core/components/Pages/NewEvent/SelectionsIncomplete/DataEntrySelectionsIncomplete.component.js:24]: I suggest adjusting to "Choose a registering unit and program to get started".

File: src/core_modules/capture-core/components/Pages/NewEvent/SelectionsIncomplete/DataEntrySelectionsIncomplete.component.js [hunk 3]
```diff
@@ -1,132 +1,52 @@
...
         } else if (!isProgramSelected) {
-            text = i18n.t('Select a program to start reporting');
+            return i18n.t('Select a program to start reporting');
```
Comment (cooper-joe) [inline src/core_modules/capture-core/components/Pages/NewEvent/SelectionsIncomplete/DataEntrySelectionsIncomplete.component.js:26]: I suggest adjusting to "Choose a program to start reporting"

File: src/core_modules/capture-core/components/Pages/NewEvent/SelectionsIncomplete/DataEntrySelectionsIncomplete.component.js [hunk 4]
```diff
@@ -1,132 +1,52 @@
...
         } else if (!isOrgUnitSelected) {
-            text = i18n.t('Select a registering unit to start reporting');
-        } else {
-            text = i18n.t('Select a category option to start reporting');
+            return i18n.t('Select a registering unit to start reporting');
```
Comment (cooper-joe) [inline src/core_modules/capture-core/components/Pages/NewEvent/SelectionsIncomplete/DataEntrySelectionsIncomplete.component.js:28]: I suggest adjusting to "Choose a registering unit to start reporting".

File: src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js [hunk 5]
```diff
@@ -202,15 +201,13 @@ const Index = ({
...
-                        <div className={classes.emptySelectionPaperContent}>
-                            {i18n.t('Make a selection to start searching')}
-                        </div>
-                    </Paper>
-                }
             </div>
+            {
+                searchStatus === searchPageStatus.INITIAL && !selectedSearchScopeId &&
+                    <InefficientSelectionsMessage
+                        message={i18n.t('Make a selection to start searching')}
```
Comment (cooper-joe) [inline src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js:208]: I suggest adjusting to "Choose a type to start searching".

**Review by cooper-joe — COMMENTED** [review]
I've added some suggested adjustments to the message texts. Seeing them all together highlighted some inconsistencies.

**Review by JoakimSM — APPROVED** [review]
Looks good 👍

**Pull request merged**